src/reducers/projects.js

```diff
@@ -25,13 +25,14 @@ const newProject = Immutable.fromJS({
     javascript: '',
   },
   enabledLibraries: new Immutable.Set(),
+  minimizedComponents: new Immutable.Set(),
```

Comment:
That's a good question. I don't think it fits as well as `enabledLibraries`, but I haven't come up with anything better either. It's probably because `minimizedComponents` really *is* a UI concern that's being crammed into the project state. Renaming `minimizedComponents` as `ui`<sup>1</sup> clarifies that the state is UI-related, but clashes with the actual `ui` state subtree and could lead to confusion.

Since I'm new to the project I'm less qualified to make this call, but I'd say given that `minimizedComponents` isn't a major part of the code base it's probably okay to leave as such, despite the cognitive dissonance.

<sup>1</sup>Idea dump: `projectUI`, `hiddenUI`, `hiddenUIComponents`, `uiState`, `editorState`, `hiddenEditors`, `editors` (this would be a whitelist instead of a blacklist)...none jump out at me as the Right answer.